WEP0000_wep-proposal.rst

```diff
@@ -0,0 +1,232 @@
+  Action: The WEP author moves the WEP moves the WEP to the ``withdrawn`` folder and merges the PR with a description of the withdrawal decision.
+
+If no community consensus was reached, the CoreOps team has the final authority to decide the resolution for a WEP. The CoreOps team is responsible to communicate their decision in the PR discussion and in community announcements as needed.
+
+Any community member can restart the WEP process on a rejected or withdrawn WEP in the future, by creating a new PR and updating the WEP according to the new information or ideas. If you are interested in restarting a WEP that was authored by someone else, we encourage you to contact the original author before you begin, in case they are interested in collaborating or have information that can help you in the process. 
+
+Implementation
+..............
+
+After a WEP is accepted, the work outlined in the WEP must be done. For most WEPs, the CoreOps team is responsible to take the needed action or coordinate with other teams to take the need actions.
+
+For example, if this WEP is accepted, it means the system is now in place and the CoreOps team will publish the information to the community website and establish the WEP team, along with other related tasks that need to be done to implement the process in the community. 
+
+Some WEPs might require specific community segments to implement the resolution, and such requirements are described in the WEPs directly. 
+
+WEP format and style
+--------------------
+
+To make the authoring and reviewing process easier, WEPs must follow a common format and style. 
+
+We use `reStructuredText <http://docutils.sourceforge.net/rst.html>`_ markup language for all WEPs, to align with the same markup that is used in all other Write the Doc documentation.
+
+In most cases, you can start by copying the `WEP template <../wep-template.rst>`_ and adding content as needed. The following sections describe the various sections of the template:
+
+
+File name
+.........
+The file name must follow the naming convention ``WEP[####]_[descriptive-but-short-title]``. For example, this WEP file is named ``WEP0000_wep-proposal``. In another example, a WEP to propose a new Slack channel naming convention might be named ``WEP0001_slack-channel-renaming``.
+  
+Metadata
+........
+The WEP file must begin with a header list of metadata fields, using the standard `RST field list <http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#field-lists>`_ format.
+
+The field list usually includes the following fields:
+
+- ``:WEP-Number:`` - Number of the WEP, in running order in 4-digit format. Only use the number value, without the acronym **WEP**. For example, this WEP is numbered ``0000``. If the number is unknown or unassigned when you first create the WEP, you can leave this field blank and the number will be updated at the resolution stage.
```

Comment:
I feel like we should probably just call this WEP0, instead of `WEP0000`. I already know that people will say `WEP1` instead of `WEP0001`, so I feel like we should probably keep them consistent with common usage.

Reply:
Looking at Python, it seems the filenames use the `0008` convention, but the actual copy is `8`: https://www.python.org/dev/peps/pep-0008/